apps/accounts/constants/defaults.py

```diff
@@ -19,7 +19,7 @@
 )
 HTK_ACCOUNTS_RESET_PASSWORD_URL_NAME = 'account_reset_password'
 HTK_ACCOUNTS_RESEND_CONFIRMATION = 'account_resend_confirmation'
-
+HTK_ACCOUNT_EMAIL_PASSWORD_CHANGED_TEMPLATE = 'accounts/password_changed'
```

Comment:
Prefer to not add more constants/HTK settings, and only do so when absolutely necessary.

In this case, we can add a default param.